hi-ml-cpath/testhisto/testhisto/mocks/tiles_generator.py

```diff
@@ -10,18 +10,22 @@
 from torchvision.utils import save_image
 
 from health_cpath.datasets.panda_tiles_dataset import PandaTilesDataset
+from health_cpath.utils.naming import TileKey
 from testhisto.mocks.base_data_generator import MockHistoDataGenerator, MockHistoDataType, PANDA_N_CLASSES
 
 
 class MockPandaTilesGenerator(MockHistoDataGenerator):
     """Generator class to create mock tiles dataset on the fly. The tiles are positioned randomly in a wsi grid."""
 
-    def __init__(self, img_size: int = 224, **kwargs: Any) -> None:
+    def __init__(self, img_size: int = 224, version: int = 0, **kwargs: Any) -> None:
         """
         :param img_size: The whole slide image resolution, defaults to 224.
+        :param version: The version of the tiles dataset, defaults to 0. This is used to support both the old and new
```

Comment:
Why do we still support the old tiling scheme?

Reply:
Not sure tbh, I think it was mainly to be able to work with old datasets that were created with the old tiling script
Also in deepmil we support both 
<img width="455" alt="image" src="https://user-images.githubusercontent.com/37396332/185412901-d23f746c-1ed0-4448-9298-e48b7dfba17d.png">